src/game_inventory.cpp

```diff
@@ -522,6 +522,21 @@ class comestible_inventory_preset : public inventory_selector_preset
                 }
             }, _( "JOY" ) );
 
+            append_cell( []( const item_location & loc ) {
+                const int health = loc->is_comestible() ? loc->get_comestible()->healthy : 0;
+                if( health > 0 ) {
+                    return "<good>healthy</good>";
```

Comment:
Please wrap these strings except the empty one with `_()` to make them translated in UI.